src/main/java/step3/domian/LottoWallet.java

```diff
@@ -0,0 +1,33 @@
+package step3.domian;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoWallet {
+
+    public final List<Lotto> lottos;
+
+    public LottoWallet(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public LottoResult compareWithLastLotto(LastWinner lastWinner) {
+        return new LottoResult(this.lottos.stream().map(lotto -> {
```

Comment:
```suggestion
        return new LottoResult(this.lottos
        .stream()
        .map(lotto -> {
                    int count = lastWinner.countSameNumber(lotto);
                    return Prize.getPrize(count, checkBonus(count, lastWinner, lotto));
                })
```
점 하나당 한줄로 사용하면 좋을거 같네요 :)
추가로 map 내부에 들어가는 등수를 판단하는 로직은 별도 메서드로 빼도 좋을거 같아요 :)